apps/emqx_utils/src/emqx_variform.erl

```diff
@@ -88,18 +90,24 @@ eval_as_string(Expr, Bindings, _Opts) ->
             {error, #{exception => C, reason => E, stack_trace => S}}
     end.
 
-%% Force the expression to return binary string.
+%% Force the expression to return binary string (in most cases).
 return_str(Str) when is_binary(Str) -> Str;
 return_str(Num) when is_integer(Num) -> integer_to_binary(Num);
 return_str(Num) when is_float(Num) -> float_to_binary(Num, [{decimals, 10}, compact]);
 return_str(Atom) when is_atom(Atom) -> atom_to_binary(Atom, utf8);
+%% For usage by other modules (e.g.: message transformation)
+return_str({?SKIP_STRINGIFICATION, X}) ->
```

Comment:
Nit: shouldn't be that some kind of `render` function flag? Otherwise we invalidate many function names, like `eval_as_string` and the very this one 🤔

Reply:
:heavy_check_mark: